### ISSUE TITLE
 missing nodelet dependency

### ISSUE THREAD

**Issue by jack-oquin** [ros-drivers/openni_launch#10]
**roslaunch_add_file_check()** complains about it when processing a package that depends upon this one:

```
FAILURE:
[/home/joq/ros/wet/bwi/src/segbot/segbot_sensors/launch/kinect/kinect.launch]:
    Missing manifest dependencies: openni_launch/manifest.xml: nodelet
```

cross-ref: utexas-bwi/segbot#20

The fix is trivial. I'll submit a pull request for it, if you want.

**Comment by jack-oquin** [comment]
Please close this when the fix has been released, so I'll know to remove the work-around from dependent packages.

**Comment by jack-oquin** [comment]
This is still awaiting release to Hydro.

Should I do the release myself?

**Comment by jonbinney** [comment]
Yeah, you can go ahead and release. Sorry I won't have time to get to it for a while.

**Comment by jack-oquin** [comment]
To do that, I need push access to the `ros-gbp/openni_launch-release` repository.

Alternatively, I could move it to the `ros-drivers-gbp` organization, where it probably belongs, anyway.

Which would you prefer?

**Comment by jack-oquin** [comment]
ref: #15

**Comment by chadrockey** [comment]
@jack-oquin I added you to openni_launch-release.  (and added an openni maintainers group).

**Comment by jack-oquin** [comment]
Fixed in openni_launch-1.9.5, just released.
